portal/src/main/webapp/js/src/DataProxyFactory.js

```diff
@@ -71,7 +68,7 @@ var DataProxyFactory = (function()
 			// last resort: send the actual case list as a long string
 			else
 			{
-				servletParams.caseList = QuerySession.getSampleIds();
+				servletParams.caseList = caseList;
```

Comment:
we can do something like this: `servletParams.caseList = caseList || QuerySession.getSampleIds();`